docs/supported-platforms.md

```diff
@@ -6,29 +6,45 @@ nav_order: 8
+* [Exoscale] (`exoscale`) - Ignition will read its configuration from the instance userdata. Cloud SSH keys are handled separately.
 * [Google Cloud] (`gcp`) - Ignition will read its configuration from the instance metadata entry named "user-data". Cloud SSH keys are handled separately.
+* [IBM Cloud] (`ibmcloud`) - Ignition will read its configuration from the instance userdata. Cloud SSH keys are handled separately.
 * Bare Metal (`metal`) - Use the `ignition.config.url` kernel parameter to provide a URL to the configuration. The URL can use the `http://`, `https://`, `tftp://`, `s3://`, or `gs://` schemes to specify a remote config.
+* [OpenStack] (`openstack`) - Ignition will read its configuration from the instance userdata via either metadata service or config drive. Cloud SSH keys are handled separately.
 * [Equinix Metal] (`packet`) - Ignition will read its configuration from the instance userdata. Cloud SSH keys are handled separately.
+* [IBM Power Systems Virtual Server] (`powervs`) - Ignition will read its configuration from the instance userdata. Cloud SSH keys are handled separately.
 * [QEMU] (`qemu`) - Ignition will read its configuration from the 'opt/com.coreos/config' key on the QEMU Firmware Configuration Device (available in QEMU 2.4.0 and higher).
 * [VMware] (`vmware`) - Use the VMware Guestinfo variables `ignition.config.data` and `ignition.config.data.encoding` to provide the config and its encoding to the virtual machine. Valid encodings are "", "base64", and "gzip+base64". Guestinfo variables can be provided directly or via an OVF environment, with priority given to variables specified directly.
+* [Vultr] (`vultr`) - Ignition will read its configuration from the instance userdata. Cloud SSH keys are handled separately.
 * [zVM] (`zvm`) - Ignition will read its configuration from the reader device directly. The vmur program is necessary, which requires the vmcp and vmur kernel module as prerequisite, and the corresponding z/VM virtual unit record devices (in most cases 000c as reader, 000d as punch) must be set online.
 
 Ignition is under active development, so this list may grow over time.
 
 For most cloud providers, cloud SSH keys and custom network configuration are handled by [Afterburn] or by platform-specific agents.
 
+[Alibaba Cloud]: https://www.alibabacloud.com/product/ecs
 [Amazon Web Services]: https://aws.amazon.com/ec2/
 [Microsoft Azure]: https://azure.microsoft.com/en-us/services/virtual-machines/
 [Microsoft Azure Stack]: https://azure.microsoft.com/en-us/overview/azure-stack/
+[BrightBox]: https://www.brightbox.com/cloud/servers/
+[CloudStack]: https://cloudstack.apache.org/
 [DigitalOcean]: https://www.digitalocean.com/products/droplets/
+[Exoscale]: https://www.exoscale.com/compute/
 [Google Cloud]: https://cloud.google.com/compute
+[IBM Cloud]: https://www.ibm.com/cloud/vpc
+[OpenStack]: https://www.openstack.org/software/releases/xena/components/nova
```

Comment:
Feels like just `https://www.openstack.org/` would be sufficient, no? This avoids linking to a specific version that'll become stale.